packages/api/test/e2e/promise-queries.spec.ts

```diff
@@ -199,48 +199,51 @@ describe.skip('Promise e2e queries', () => {
     );
   });
 
-  describe('with plain type', () => {
-    const EXISTENTIAL_DEPOSIT = 500;
-    it('queries correct value', async () => {
-      const existentialDeposit = await api.query.balances.existentialDeposit() as Balance;
+  // @TODO: existentialDeposit has been moved to consts; rewrite the following
```

Comment:
Should we do like in Rust: never commit commented code?

If there's commented code, create an issue with this code.

Reply:
yes, sounds good